app/views/outcomes-report/OutcomesReportResultComponent.vue

```diff
@@ -90,13 +90,21 @@ export default Vue.extend({
         course.studentsCompleting = (this.org.progress.studentsCompletingCourse || {})[course._id] || 0
         course.completion = course.studentsStarting ? Math.min(1, course.studentsCompleting / course.studentsStarting) : 0
         if (this.org.newProgress) {
-          const maxCompleteLevels = Math.max(Object.values(this.org.newProgress.courseCompleteLevels[course._id] || {})) || 0
-          course.completion = Math.min(1, maxCompleteLevels / this.org.newProgress.courseAllLevels[course._id])
+          const courseCompleteLevels = this.org.newProgress.courseCompleteLevels || {}
+          const courseCompleteStudents = this.org.newProgress.courseCompleteStudents || {}
+          const completeLevels = Math.max(...Object.values(courseCompleteLevels[course._id] || {})) || 0
+          course.studentsStarting = courseCompleteStudents[course._id] || 0
```

Comment:
what does this field indicate? These are the students who have completed levels not just started, correct?

Reply:
yes, we need this filed to show the total number of students who starts the course, since in html doms it already use this name in old reports, i didn't change the property name

Reply:
studentsStarting and courseComplete will be different, right? Is this calculation intentional?

Reply:
haha good catch. rename it to courseStartingStudents. i don't know why i name it complete 😂